fullscreen.bs

```diff
@@ -314,6 +314,10 @@ attribute's getter must run these steps:
 <ol>
  <li><p>Let <var>promise</var> be a new promise.
 
+ <li><p>If <var>doc</var> has no <a for=Document>browsing context</a>, or if <var>doc</var>'s
+ <a for=Document>browsing context</a>'s <a>active document</a> is not <var>doc</var>, then
```

Comment:
I think this needs to be `if <var>doc</var> is not <a>fully active</a>`, since it might be the active document but an ancestor browsing context's document is not active.